lxa_iobus/server/server.py

```diff
@@ -139,6 +150,14 @@ async def flash_worker(self):
                 logger.exception('flashing failed')
 
     # views ###################################################################
+    async def get_server_info(self, request):
+        return json_response({
+            'hostname': os.uname()[1],
+            'started': str(self.started),
+            'can_interface': self.network.interface,
+            'can_interface_is_up': self.network.interface_is_up(),
```

Comment:
I really like the interface information on the website.

One idea here:
```
Can Interface: can0_iobus
Can Interface State: UP
```

An interesting fact about can are the error counters and a specific link state:
```
$ ip -detail l show dev can0_iobus
8: can0_iobus: <NOARP,UP,LOWER_UP,ECHO> mtu 16 qdisc pfifo_fast state UP mode DEFAULT group default qlen 10
    link/can  promiscuity 0 minmtu 0 maxmtu 0 
    can state ERROR-ACTIVE restart-ms 10000 
          bitrate 100000 sample-point 0.875 
          tq 625 prop-seg 6 phase-seg1 7 phase-seg2 2 sjw 1
          gs_usb: tseg1 1..16 tseg2 1..8 sjw 1..4 brp 1..1024 brp-inc 1
          clock 48000000numtxqueues 1 numrxqueues 1 gso_max_size 65536 gso_max_segs 65535 
```

`state ERROR-ACTIVE` is the relevant part here. would it be useful to show this in the front end? (Maybe even in a later release?) @lichtfeind @hnez

Reply:
After some discussion: We will postpone this into a later release.